tests/atest/rules/spacing/misaligned-continuation-row/test.robot

```diff
@@ -0,0 +1,58 @@
+*** Settings ***
+Documentation      Here we have documentation for this suite.
+...                Documentation is often quite long.
+...
+...               It can also contain multiple paragraphs.
+Default Tags       default tag 1    default tag 2    default tag 3
+...                default tag 4    default tag 5
+Force Tags
+...   tag
+...    tag2
+
+
+*** Variable ***
+${STRING}          This is a long string.
+...                It has multiple sentences.
+...                It does not have newlines.
+${MULTILINE}       SEPARATOR=\n
+...                This is a long multiline string.
+...               This is the second line.
+...                This is the third and the last line.
+@{LIST}            this     list     is      quite    long     and
+...                 items in it can also be long
```

Comment:
I think this one should also be recognized as a violation of the rule.

Reply:
I have problem with this one. I'm thinking how we should process it. In this case it's simple:
```
My Keyword    ${arg}
...  1
...  2
...  3
```
I definietly want to have ... 1, 2, 3 aligned. But what about ${arg}? if I would check for alignment with first line (like in linked example with LIST) it would look like:
```
My Keyword    ${arg}
...           1
...           2
...           3
```
So in this case I think it's better to not align with first line. So how about ignoring first line for keywords etc but checking first line for variables only?:
```
@{LIST}            this     list     is      quite    long     and
...                items in it can also be long
```

Reply:
Same for [Tags], [Arguments] etc - they should be aligned with first row

Reply:
> So how about ignoring first line for keywords etc but checking first line for variables only?

I agree.